README.md

```diff
@@ -1,25 +1,121 @@
 # middie
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/fastify/middie.svg)](https://greenkeeper.io/)
-
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/) [![Build Status](https://travis-ci.org/fastify/middie.svg?branch=master)](https://travis-ci.org/fastify/middie)
 
-*middie* is the module that add middlewares support on steroids to [Fastify](https://www.npmjs.com/package/fastify).
+*middie* is the plugin that add middlewares support on steroids to [Fastify](https://www.npmjs.com/package/fastify).
 
 The syntax style is the same as [express](http://npm.im/express)/[connect](https://www.npmjs.com/package/connect).
 Does not support the full syntax `middleware(err, req, res, next)`, because error handling is done inside Fastify.
 
-If you want to see how use this module with Fastify, check [here](https://github.com/fastify/fastify/#fastifyusemiddlewarereq-res-next).
-
 ## Install
 
 ```
-npm install middie --save
+npm install middie
```

Comment:
why not `--save`?

Reply:
Because from npm@5 is the default operation ;)